src/app/shared/services/filter-data.service.ts

```diff
@@ -14,13 +12,55 @@ export class FilterDataService {
     'goals-pending': [],
     'task-summary': [],
     'member-summary': [],
-    };
+  };
+  coloumData = {};
+  constructor() { }
 
+  setColumnData(retroId, tableKey, data) {
+    if (!this.coloumData[retroId]) {
+      this.coloumData[retroId] = {
+        'key-takeaways': [],
```

Comment:
These keys are used in both filters and columns, we should make constants for them (I believe we already have their constants).

Reply:
changed